pumps/mongo_aggregate.go

```diff
@@ -170,12 +177,32 @@ func (m *MongoAggregatePump) Init(config interface{}) error {
 	if m.dbConf.ThresholdLenTagList == 0 {
 		m.dbConf.ThresholdLenTagList = THRESHOLD_LEN_TAG_LIST
 	}
+	// if StoreAnalyticsPerMinute is set to true, the aggregation time will be set to 1.
+	// if not, the aggregation time will be set to the value of the field AggregationTime.
+	// if there is no value for AggregationTime, it will be set to 60.
+	if m.dbConf.StoreAnalyticsPerMinute {
+		m.log.Info("StoreAnalyticsPerMinute is set to true. Setting AggregationTime to 1.")
```

Comment:
lets be more informative here. Lets say something like: `Pump will aggregate data every minute.` or similar.